R/LSTbinomDist.R

```diff
@@ -0,0 +1,53 @@
+#
+# Copyright (C) 2019 University of Amsterdam
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+LSTbinomDist <- function(jaspResults, dataset, options) {
+  
+  options <- jaspDistributions:::.ldRecodeOptionsBinomial(options)
```

Comment:
In any case, you will need to add `jaspDistributions` to the `DESCRIPTION` file.